### DISCUSSION TITLE
DockerHubManager.php: Could notcreate container...

### DISCUSSION THREAD

**Discussion by surcouf** [nextcloud/all-in-one#4826] [Questions] [answered]
### Steps to reproduce
1. Start docker image nextcloud-aio:latest 
2.  curl -sS https://registry-1.docker.io/v2/nextcloud/aio-domaincheck/manifests/5000/nextcloud/all-in-one
404 page not found

#### Other valuable info 
NOTICE: PHP message: Could not get digest of container nextcloud/aio-domaincheck:5000/nextcloud/all-in-one Client error: `HEAD https://registry-1.docker.io/v2/nextcloud/aio-domaincheck/manifests/5000/nextcloud/all-in-one` resulted in a `404 Not Found` response
NOTICE: PHP message: Not pulling the image for the nextcloud/aio-domaincheck container because docker hub does not seem to be reachable.
NOTICE: PHP message: Could not start domaincheck container: Could not create container nextcloud-aio-domaincheck: Client error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-domaincheck` resulted in a `400 Bad Request` response:
{"message":"invalid reference format"}
NOTICE: PHP message: Could not get digest of container nextcloud/all-in-one:5000/nextcloud/all-in-one Client error: `HEAD https://registry-1.docker.io/v2/nextcloud/all-in-one/manifests/5000/nextcloud/all-in-one` resulted in a `404 Not Found` response
NOTICE: PHP message: Could not get digest of container nextcloud/aio-domaincheck:5000/nextcloud/all-in-one Client error: `HEAD https://registry-1.docker.io/v2/nextcloud/aio-domaincheck/manifests/5000/nextcloud/all-in-one` resulted in a `404 Not Found` response
NOTICE: PHP message: Not pulling the image for the nextcloud/aio-domaincheck container because docker hub does not seem to be reachable.
NOTICE: PHP message: Could not start domaincheck container: Could not create container nextcloud-aio-domaincheck: Client error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-domaincheck` resulted in a `400 Bad Request` response:
{"message":"invalid reference format"}
NOTICE: PHP message: Could not get digest of container nextcloud/all-in-one:5000/nextcloud/all-in-one Client error: `HEAD https://registry-1.docker.io/v2/nextcloud/all-in-one/manifests/5000/nextcloud/all-in-one` resulted in a `404 Not Found` response

**Comment by szaimen** [comment, marked as answer]
> because docker hub does not seem to be reachable.

This is your issue. So it looks like the container does not have access to the internet or docker hub is down
